x/slashing/spec/05_hooks.md

```diff
@@ -5,6 +5,18 @@ order: 5
 # Hooks
 
 In this section we describe the "hooks" - slashing module code that runs when other events happen.
+The module provides the following hooks, which can be useful
```

Comment:
Maybe I'd rephrase both lines (this and the one above) as follows:

```
This section contains a description of the module's `hooks`. Hooks are operations that are executed automatically when events are raised.
```

Reply:
OK, looks good